src/lang/english.txt

```diff
@@ -5806,10 +5806,10 @@ STR_TOWN_NAME                                                   :{TOWN}
 STR_VEHICLE_NAME                                                :{VEHICLE}
 STR_WAYPOINT_NAME                                               :{WAYPOINT}
 
-STR_CURRENCY_SHORT_KILO                                         :{NBSP}k
-STR_CURRENCY_SHORT_MEGA                                         :{NBSP}m
-STR_CURRENCY_SHORT_GIGA                                         :{NBSP}bn
-STR_CURRENCY_SHORT_TERA                                         :{NBSP}tn
+STR_CURRENCY_SHORT_ABBREVIATIONS_PER_POWER_OF_TEN               :3={NBSP}k|6={NBSP}m|9={NBSP}bn|12={NBSP}tn|15={NBSP}Qa|18={NBSP}Qi
```

Comment:
Should this be a language attribute rather than trying to force it into a string?

Reply:
Is that attribute going to have a significantly different format? It would likely mean parsing the string in strgen, then encoding it somehow and decoding is in OpenTTD. That seems like way more work overall.

Does the translator support changing attributes? If not, who's going to add that, or figure out the separators for all languages?

Reply:
We also don't have a way to explain to the translators what this format means. So that is something to consider. Having it as language properties might help, as they have to change that via us. Or we could look at a way to add hints in eints based on a string.

Reply:
Hints to eints might be a good idea,  even generically for some strings which could use so more information when translating. However, eints is quite far out of my comfort zone and honestly I do not want this feature that bad that I'm going to spend lots of time learning eints and its intricacies.

I think the number formats could fairly easily be attributes, as I doubt there are significantly many more translated languages than the CJK and Hindi. And the separators for those are already mentioned in the comments of this PR.

The abbreviations are way trickier, as those will essentially be different for each language and that means likely over 50 requests for changing the properties.

Reply:
> that I'm going to spend lots of time learning eints and its intricacies.

Please don't. But good news for you: you have a bunch of people around you that can help out! So if we think that is the best way to solve this problem, we can make that happen :) But it first requires knowing that is actually the best way to approach it ;) But please don't feel bad you don't know how to modify eints, or spend your time learning it if you don't want to. It really isn't a problem :)

(I still think attributes is the better solution, so at the moment I just await what form will eventually be chosen by this PR :) But if I turn out to be placing my eggs in the wrong basket, just let us know!)

Reply:
Eints will preserve `##attributes`, which it does not know (like `##decimalsep`).
No problem in adding more of those.